lua/modules/menu.lua

```diff
@@ -100,6 +101,19 @@ function closeModal()
 	end
 end
 
+-- pops active modal content
+-- closing modal when reaching root content
+function popModal()
+	if activeModal == nil then
+		return
+	end
```

Comment:
Wouldn't it be better to call `closeModal` here? `if activeModal ~= nil and #activeModal.contentStack > 1 then ...`

Reply:
If activeModal is nil, there's nothing to close or pop.
I like the if + early return design for clarity.